### DISCUSSION TITLE
Documentation search: show all results is broken

### DISCUSSION THREAD

**Discussion by drjasonharrison** [SigNoz/signoz#1288] [Q&A]
When I use the search field at https://signoz.io/docs/ and search for "refresh" there are four results listed. When I click on "See all 4 results" the page https://signoz.io/search/?q=refresh is loaded. Unfortunately it tells me:

"No results were found"

I can click on any of the results in the "pop up dialog" when I start the search from the field, but the search page results are broken.

**Comment by pranay01** [comment]
thats weird. Let me check and get back. 
We use Algolia for search. So will have to check if there are any issues with the config
